bin/tictactoe

```diff
@@ -1,3 +1,28 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+require_relative '../lib/game.rb'
+
+puts "Welcome to Tic-Tac-Toe!  Please make a selection:"
+puts "1. 0 player game (2 computers)"
+puts "2. 1 player game (play against the computer)"
+puts "3. 2 player game (play against a friend)"
+
+input = gets.chomp
+
+
+case input
+when '1'
+  game = Game.new(player_1 = Computer.new('X'), player_2 = Computer.new('O'))
+  game.play
+when '2'
+  player_1 = Human.new('X')
+  player_2 = Computer.new('O')
+  game = Game.new(player_1, player_2)
+  game.play
+when '3'
+  game = Game.new
+  game.play
+else
+  puts "Incorrect input."
+end
```

Comment:
Prefer to encapsulate CLI logic into a class. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.